images/prow/cloudbuild.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 1200s
+options:
+  substitution_option: ALLOW_LOOSE
```

Comment:
nit/question:
Why do we need that?
I'ts probably c&p, but still I'd like to understand why we want to explicitly enable this loose setting. Do you know why?

Reply:
I have no idea I took this setting from the k8s-cloud-builder one. :man_shrugging: